usage.py

```diff
@@ -87,6 +87,15 @@
 
         spacer,
 
+        html.Div([
+            html.P([html.Strong('Test for Snackbar')]),
+            html.Button('Open/Close Snackbar', id='snackbar-button'),
+            sd_material_ui.Snackbar(id='snackbar',
+                                    message='You opened the Snackbar!')
+        ]),
```

Comment:
Would you also add an example of using the `action` inside the snackbar?

Reply:
Should I just use an 'X' here?

Reply:
![image](https://user-images.githubusercontent.com/47650493/90437241-09154700-e0a0-11ea-9121-8cd60b7ff3fd.png)

Reply:
That looks fine. Does it close the snackbar before the timer otherwise would?

Reply:
No, and I can't pass in a button component

Reply:
Description for action: `The action to display. It renders after the message, at the end of the snackbar.`

Reply:
Ok. If the action isn't able to perform a function, then let's drop it so there's no confusion.

Reply:
Ok, good to merge now @coralvanda